### ISSUE TITLE
Gnome Terminal white scrollbar

### ISSUE THREAD

**Issue by Corsseir** [system76/pop-gtk-theme#40]
**NOTE: Before reporting a bug, please check whether it also occurs with other themes.**
> If the bug also occurs with other themes (especially upstream default theme),
it is very likely an upstream issue.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
**Reporter info**

```
 * Distribution - (run ```lsb_release -a```)
 * Gtk+ 3.0 version - (run ```apt policy libgtk-3-0```)
 * Desktop environment - 
 * Pop version - (run ```apt policy pop-gtk-theme```)
 ```

**Related Application and/or Shell theming (or name of widget)**
gnome-terminal

**Actual issue**
Gnome Terminal white scrollbar.

**Steps to reproduce**
Just open gnome-terminal.

**Expected behavior**
Scrollbar adjusted to the pop gtk theme.

**Behavior in Adwaita**
Excellent

**Other Note (feature-request, question, etc...)**
![6](https://user-images.githubusercontent.com/11137173/31138727-ba5a691c-a86f-11e7-810e-6d39dce2f4a6.png)

**Comment by isantop** [comment]
This is happening because the terminal is using the light theme, which should rightly use the white-colored titlebar. Probably the best way to fix this for system76/pop-default-settings to set the Dark theme variant by default, since this also fixes the white tab bar when tabs are open.

**Comment by jackpot51** [comment]
https://github.com/system76/pop-default-settings/issues/21

**Comment by isantop** [comment]
Fixed in 38d43079baf6b0ba25816c928a60b2379ca1b337
